packages/vue/src/components/authenticator.vue

```diff
@@ -144,126 +144,6 @@ const signInLabel = computed(() => getSignInTabText());
 const createAccountLabel = computed(() => getSignUpTabText());
 
 // methods
-const onSignInSubmitI = (e: Event) => {
```

Comment:
Github won't let me comment on the line but we also want to remove the `emit` object as well on line 64. Looks like you got the rest of them in the other files